.github/workflows/publish-to-test-pypi.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+
+on: push
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Build a binary wheel and a source tarball
+      run: python setup.py sdist bdist_wheel
```

Comment:
I always did `sdist` only and did it with a zip format

Reply:
so just `python setup.py sdist` ?